_data/advisories.yaml

```
@@ -11,8 +11,7 @@ texts:
   experimental: "The functionality described on this page is marked as Experimental, and as such, may change before it becomes generally available."
   rc: "The Swarm mode feature included in Docker Engine 1.12 is a release candidate feature and might be subject to non backward-compatible changes. Some functionality may change before the feature becomes generally available."
   docker4mac-beta: "Docker Desktop for Mac is currently in public beta. Some functionality may change before the product becomes generally available."
-  docker4win-beta: "Docker Desktop for Windows is currently in public beta. Some functionality may change before the product becomes generally available."
-  toolbox: "**Legacy desktop solution.** Docker Toolbox is for older Mac and Windows systems that do not meet the requirements of [Docker Desktop for Mac](/docker-for-mac/) and [Docker Desktop for Windows](/docker-for-windows/). We recommend updating to the newer applications, if possible."
+  docker4win-beta: "Docker Desktop for Windows is currently in public beta. Some functionality may change before the product becomes generally available.
```

I think you accidentally removed a quote (guess that's what it's failing on);

```
8:03:40 PM: jekyll 3.8.5 | Error:  (/opt/build/repo/_data/advisories.yaml): did not find expected alphabetic or numeric character while scanning an alias at line 15 column 15
```

```suggestion
  docker4win-beta: "Docker Desktop for Windows is currently in public beta. Some functionality may change before the product becomes generally available."
```